### PR TITLE
Load instantsearch asynchronously

### PR DESCRIPTION
**Pull request by staabm**
untested change, waiting for the PR patch deployment to verify.

### CODE REVIEW

File: res/views/layout.html.twig [hunk 1]
```diff
@@ -87,7 +87,7 @@
         <script src="/dist/js/main.min.js?v={{ version }}"></script>
 
         <script src="https://cdn.jsdelivr.net/algoliasearch/3/algoliasearch.min.js"></script>
-        <script src="https://cdn.jsdelivr.net/instantsearch.js/1/instantsearch.min.js"></script>
+        <script src="https://cdn.jsdelivr.net/instantsearch.js/1/instantsearch.min.js" async></script>
```
Comment (staabm) [inline res/views/layout.html.twig]: this is the very last file in the algolia-request chain to be loaded, therefore can be loaded asynchronously (after algoliasearch.min successfully loaded synchronously)

Comment (mnapoli) [reply res/views/layout.html.twig]: That makes sense indeed, can I merge?

Comment (staabm) [reply res/views/layout.html.twig]: I guess the patch is complete, but atm untested

**Comment by staabm** [comment]
TODO

as we learned with https://github.com/mnapoli/externals/pull/81 , there is some in-page js which needs to be fired after this lib loaded. since we make it async here, we need to wire it with an unload callback handler to prevent a "undefined instasearch" symbol error

https://github.com/mnapoli/externals/blob/6f58221d18e8ddc81353eb310c50731e69eb2dad/res/views/home.html.twig#L73-L106

**Comment by mnapoli** [comment]
Just to be sure I understand it right: I should wait before merging that?

**Comment by staabm** [comment]
I feel this is ready (but still untested)

**Comment by mnapoli** [comment]
Let's give it a try

**Pull request merged**